src/interfaces/IPOOL.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+interface IPOOL {
```

Comment:
*(arbitrary comment)*

Same as gitcoin with a few methods missing

Reply:
Did you verify this with `cast interface` on the actual POOL deployment?

Reply:
Oh I didn't know that was a thing. I verified visually

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/9/commits/22bfdac495003fc5a43b86e3c0d63226828a25c6

Reply:
It's a great feature!